DefaultTabBar.js

```diff
@@ -20,6 +20,8 @@ const DefaultTabBar = React.createClass({
     inactiveTextColor: React.PropTypes.string,
     textStyle: Text.propTypes.style,
     tabStyle: View.propTypes.style,
+    underlineStyle: View.propTypes.style,
+    tabBarPosition: React.PropTypes.oneOf(['top', 'bottom', 'overlayTop', 'overlayBottom', ]),
```

Comment:
```tabBarPosition``` I would prefer remove this property. It is pretty straightforward to achieve it using ```underlineStyle```.